content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,333 @@ Download the Istio release which includes installation files, samples, and the
-    kiali                       ClusterIP      172.21.192.82    <none>           20001/TCP                                                                                                                    11s
-    prometheus                  ClusterIP      172.21.30.218    <none>           9090/TCP                                                                                                                     11s
-    tracing                     ClusterIP      172.21.126.166   <none>           80/TCP                                                                                                                       11s
-    zipkin                      ClusterIP      172.21.122.59    <none>           9411/TCP                                                                                                                     11s
+    $ kubectl get pods
+    NAME                              READY   STATUS            RESTARTS   AGE
+    details-v1-78d78fbddf-tj56d       0/2     PodInitializing   0          2m30s
+    productpage-v1-85b9bf9cd7-zg7tr   0/2     PodInitializing   0          2m29s
+    ratings-v1-6c9dbf6b45-5djtx       0/2     PodInitializing   0          2m29s
+    reviews-v1-564b97f875-dzdt5       0/2     PodInitializing   0          2m30s
+    reviews-v2-568c7c9d8f-p5wrj       1/2     Running           0          2m29s
+    reviews-v3-67b4988599-7nhwz       0/2     PodInitializing   0          2m29s
     {{< /text >}}
 
     {{< tip >}}
-    If your cluster runs in an environment that does not
-    support an external load balancer (e.g., minikube), the
-    `EXTERNAL-IP` of `istio-ingressgateway` will display
-    `<pending>`. To access the gateway, use the service's
-    `NodePort`, or use port-forwarding instead.
+    Re-run the previous command and wait until all pods report READY 2 / 2 and
+    STATUS Running before you go to the next step. This might take a few minutes
+    depending on your platform.
     {{< /tip >}}
 
-    Also ensure corresponding Kubernetes pods are deployed and have a `STATUS` of `Running`:
+1.  Verify everything is working correctly up to this point. Run this command to
+    see if the app is running inside the cluster and serving HTML pages by
+    checking for the page title in the response:
+
+    {{< text bash >}}
+    $ kubectl exec -it $(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}') -c ratings -- curl productpage:9080/productpage | grep -o "<title>.*</title>"
+    <title>Simple Bookstore App</title>
+    {{< /text >}}
+
+## Open the application to outside traffic {#ip}
+
+Your app is deployed but not accessible from the outside. To make it accessible,
+you need to create an
+[Istio Ingress Gateway](/docs/concepts/traffic-management/#gateways), which maps a path to a
+route at the edge of your mesh.
+
+1.  Connect this application to be published by the Istio gateway:
```

Comment:
Grammer..  c/by/to might be enough to make it sound OK.

Reply:
fixed